src/turn.js

```diff
@@ -0,0 +1,29 @@
+class Turn {
+  constructor(userGuess, card) {
+    this.userGuess = userGuess;
+    this.card = card;
+  }
+
+  returnGuess() {
+    return this.userGuess;
+  }
+
+  returnCard() {
+    return this.card;
+  }
+
+  evaluateGuess() {
+    return (this.userGuess === this.card['correctAnswer']);
+  }
+  
+  giveFeedback() {
+    let feedback = this.evaluateGuess();
+    if (feedback) {
```

Comment:
This is a good use case for a ternary if you want to save some lines, although the if/else works just fine